.github/workflows/scheduled.yaml

```diff
@@ -0,0 +1,49 @@
+# Tests in this workflow are run periodically.
+# They are generally very expensive and not on the critical path
+# for development
+# scheduled workflows only run in the default branch
+
+name: Scheduled Tests
+
+on:
+  schedule:
+    - cron: "42 3 * * 0"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  coverity-scan:
+    name: Scan with Coverity
+    # only run the workflow on Squid's main repository
+    if: github.repository == 'squid-cache/squid'
+
+    runs-on: ubuntu-22.04
+    # this job only runs if the github repository contains username
+    # and password to access the Coverity Scan service
+    env:
+      coverity_user: ${{ secrets.COVERITY_USER }}
+      coverity_token: ${{ secrets.COVERITY_TOKEN }}
+
+    container:
+      image: squidcache/buildfarm-coverity:stable
+      options: --user 1001 # uid used by worfklow runner
+
+    steps:
+      - name: Checkout Sources
+        if: env.coverity_user && env.coverity_token
+        uses: actions/checkout@v4
+
+      - name: Scan
+        if: env.coverity_user && env.coverity_token
```

Comment:
apply condition: do not run the scan unless the secrets are defined.
Due to Github Actions restrictions, it's not possiblte to specify this condition at the
job level, but only at the step level

Reply:
removed